examples/multimodal_v1/components/worker.py

```
@@ -0,0 +1,290 @@
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import copy
+import asyncio
+import logging
+import os
+import signal
+import socket
+from typing import Optional, AsyncGenerator
```

Ruff (0.12.2)

`typing.AsyncGenerator` imported but unused

Remove unused import: `typing.AsyncGenerator`

(F401)